### PR TITLE
Update script to import admins

### PR DESCRIPTION
**Pull request by sanikasudhalkar**

### CODE REVIEW

**Comment by sanikasudhalkar** [comment]
@nblair @aghadiri

File: iq-apply-conf.py [hunk 1]
```diff
@@ -68,6 +68,9 @@ def main():
     # Admin level configuration and integrations
     nexus_administration(config)
 
+    # ROOT level configuration
+    root_configuration(config)
```
Comment (CSMonkee) [inline iq-apply-conf.py:72]: root_configuration is deprecated code. Please update your baseline to the latest checkin on master branch.

File: iq-apply-conf.py [hunk 2]
```diff
@@ -193,6 +197,11 @@ def post_url(url, params, root=""):
     resp = iq_session.post(url, json=params, auth=iq_auth, verify=not self_signed)
     return handle_resp(resp, root)
 
+def put_url(url, params, root=""):
```
Comment (CSMonkee) [inline iq-apply-conf.py:200]: This function already exists.

File: iq-healthcheck.py [hunk 3]
```diff
@@ -1177,11 +1192,12 @@ def set_webhooks():
...
     data = get_url(url)
-    for webhook in data:
-        webhooks[webhook['id']] = webhook['description']
+    if data is not None:
+        for webhook in data:
+            webhooks[webhook['id']] = webhook['description']
 
 
-# Write the data to a file...
+# Write the data to a file...ß
```
Comment (CSMonkee) [inline iq-healthcheck.py:1200]: Have you inadvertently added a character to this comment?

**Review by CSMonkee — COMMENTED** [review]
It appears that your changes were based upon a version of this code that is no longer the latest. Please re-base and we can merge your changes. Thanks for contributing.

**Review by CSMonkee — APPROVED** [review]
Looks good. Thanks for contributing.

**Pull request merged**